audit.go

```diff
@@ -603,20 +603,28 @@ type AuditStatus struct {
 	BacklogWaitTimeActual uint32          // Time the kernel has spent waiting while the backlog limit is exceeded.
 }
 
-const sizeofAuditStatus = int(unsafe.Sizeof(AuditStatus{}))
+const (
+	sizeofAuditStatus = int(unsafe.Sizeof(AuditStatus{}))
+
+	// MinSizeofAuditStatus is the minimum usable message size that
+	// is acceptable for unmarshaling from the wire format. Messages
+	// this size do not report backlog_wait_time_actual.
+	MinSizeofAuditStatus = sizeofAuditStatus - int(unsafe.Sizeof(uint32(0)))
```

Comment:
nit: Is there a need to export this symbol?

Reply:
It's documentational. This allows the user to know the limits of the API.